README.md

```diff
@@ -56,7 +56,7 @@ Let's start by looking at what a fully configured Store looks like. We will then
 ```kotlin
 StoreBuilder
     .from(
-        fetcher = nonFlowValueFetcher { api.fetchSubreddit(it, "10").data.children.map(::toPosts) },
+        fetcher = Fetcher.from { api.fetchSubreddit(it, "10").data.children.map(::toPosts) },
```

Comment:
`SourceOfTruth` uses `from` and `fromNonFlow` we should decide on one style

Reply:
I forgot why we don't just use `from` for everything. is it type erasure issues or naming preference?

Reply:
type erasure. even if we give them different java names, when the same function name receives different types of Functions, it cannot auto resolve in the call site (so it forces you to specify types when calling it)